resources/prometheus/prometheus-rules.yaml

```diff
@@ -454,6 +454,78 @@ spec:
             )
           record: central:sli:availability:extended_avg_over_time28d
 
+        # - Queries the 90th percentile of central's handled GRPC/HTTP API requests latencies over the last 10 minutes.
+        - expr: |
+            (histogram_quantile(0.9, sum  by(le, namespace, grpc_service, grpc_method) (rate(grpc_server_handling_seconds_bucket{container="central", grpc_method!~"ScanImageInternal|DeleteImages|EnrichLocalImageInternal|RunReport|ScanImage|TriggerExternalBackup|Ping"}[10m]))) > 0) < bool 0.1
```

Comment:
TIL, didn't know about `< bool`. Probably can use that else where as well to simplify the promQL.